### ISSUE TITLE
Resolve release block

### ISSUE THREAD

**Issue by Haehnchen** [Haehnchen/idea-php-toolbox#1]
some unorder todo list based on https://github.com/Haehnchen/idea-php-symfony2-plugin/milestones/Plugin%20Cleanup :
- migrate ui: type provider, method references
- finalize json configuration file syntax: https://github.com/Haehnchen/idea-php-toolbox/blob/master/tests/de/espend/idea/php/toolbox/utils/test.json
- publish doc: http://phpstorm.espend.de/php-toolbox
- provide view variable storage: smarty, twig, blade support
- merge symfony2 server branch to provide remote api
- annotation completion: https://github.com/Haehnchen/idea-php-symfony2-plugin/issues/249
- more array completion options

nice2have:
- provide symfony2 bundle and php lib
- basic configurations for common use cases: phpunit, sonata, ...

**Comment by Koc** [comment]
@Haehnchen any news on this plugin? Can you switch from Symfony2 plugin to this? Many issues from Symfony2 plugin can be resolved by community when plugin extension point will be added

**Comment by Haehnchen** [comment]
@Koc going next here, because of to much code duplications in serveral plugin.

**Comment by Koc** [comment]
@Haehnchen please excuse me for boring, but have you plan to release some beta version of the plugin for testing it?

**Comment by Haehnchen** [comment]
yes, shortly before :)

**Comment by Haehnchen** [comment]
@Koc uploaded

**Comment by Koc** [comment]
@Haehnchen Awesome! Can I download compiled jar before it will be approved by JetBrains team?

**Comment by King2500** [comment]
Should this plugin replace the features from my plugin "PHP Advanced AutoComplete" ?
If that's the plan, I would like to contribute the existing completions into your "default" json.
(Maybe some features are missing here, like prefixed completion for `header` completion etc)

An other idea would be, making the Toolbox plugin a requirement for other plugins like mine (PHP core, Symfony, PHPUnit, etc).

What do you think?

**Comment by Haehnchen** [comment]
yes we are all trying to resolve the same problems. that's why i am splitting my code base into smaller pieces. your plugin provides a nice completion collection, so feel free to generate some json files :)

there some more features we need: prefixed completion, php execution, symfony commands?, external urls, javascript, ...
